robot_modeling/DQ_Kinematics.m

```diff
@@ -125,6 +140,27 @@ function set_base_frame(obj, base_frame)
       
     end
     
+    methods (Access = protected)        
+        function check_to_ith_link(obj, to_ith_link)
```

Comment:
I implemented the method using the same name as in C++.  @mmmarinho do you have any comments?

Reply:
@juanjqo @bvadorno 
The local variable it was checking was called ‘to_ith_link’, first used in methods such as ‘fkm(q,to_ith_link)’ and ‘pose_jacobian(q,to_ith_link)’. These follow Bruno’s logic, because, for example, the local variable refers to the FKM up to a given link. 

The naming might not be the same as in MATLAB. 

One way or the other, it might be a good chance to unify those even though they are all private/protected  or local. 

@juanjqo  If in the C++ version the leading _ is missing on the method signature, please also add it there.